custom-connector-sdk/src/main/java/com/amazonaws/appflow/custom/connector/model/write/WriteDataRequest.java

```diff
@@ -67,7 +67,8 @@
     List<String> records();
 
     /**
-     * Write all or none if any of the entity records error out.
+     * Specifies that Write operation will fail after the first instance of a failure when attempting
```

Comment:
Specifies that the WRITE operation must fail immediately after encountering the first instance of failure when writing a batch of records to the Application. Alternatively, if the application supports the allOrNone behavior the connector can pass on the flag to the application.

Reply:
done